client/scss/utilities/_classes.scss

```diff
@@ -0,0 +1,3 @@
+.touch-scroll { // Adds momentum scrolling to iOS
+  -webkit-overflow-scrolling: touch;
```

Comment:
Nothing jumps out at me on  a quick google. The experience without this property definitely feels broken on iOS. Could be worth amending/removing if/when we add scroll snapping though. I'll add a TODO.